Util/ClassInfo.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the DunglasJsonLdApiBundle package.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Dunglas\JsonLdApiBundle\Util;
+
+use Doctrine\Common\Util\ClassUtils;
+
+/**
+ * Retrieves informations about a class.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+trait ClassInfo
+{
+    /**
+     * Get class name of the given object.
+     *
+     * @param object $object
+     *
+     * @return string
+     */
+    private function getObjectClass($object)
```

Comment:
It is used several times and instanstiating an object is more expensive than a trait.
Last but not least: I :heart: traits.